core/base/common/VisitedMask.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
```

Comment:
thanks for adding your name, email address and date.

Reply:
Fixed